src/SystemInternal.hh

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  *
 */
-#ifndef IGNITION_GAZEBO_SYSTEMINTERNAL_HH_
-#define IGNITION_GAZEBO_SYSTEMINTERNAL_HH_
+#ifndef GZ_GAZEBO_SYSTEMINTERNAL_HH_
```

Comment:
Can the header guards be `GZ_SIM`?

Reply:
Not until the CMake project name becomes `XXX-sim`

Reply:
https://github.com/gazebosim/gz-sim/pull/1496/commits/465b1796de4dc10cb154a369892cd7582743f4e2